modules/standard/Errors.chpl

```diff
@@ -275,7 +275,7 @@ module Errors {
     proc deinit() {
       if errorsArray {
         for i in 0..#nErrors {
-          errorsArray[i].clear();
+          delete owned.release(errorsArray[i]);
```

Comment:
Not related to this PR, but I could imagine potential problems if a user failed to `delete` the `owned.release()`, imagining it to give up the ownership without returning anything / requiring any further action.  We've discussed before a (general) warning for returning procedures/methods whose return values are ignored by the user, and that could help in this case as well.  But I'm betting that someone will get bitten by this before we implement that.

Reply:
I believe we suggested to replace clear() with nil assignment. Here it should be `errorsArray[i] = nil;`